cairocffi/test_xcb.py

```diff
@@ -149,6 +155,11 @@ def test_xcb_pixmap(xcb_conn):
         width, height
     )
 
+    ctx = None
+    surface = None
+    remove_gc(xcb_conn, gc)
+    remove_pixmap(xcb_conn, pixmap)
```

Comment:
The library has the ffi.gc methods for the surface and context set, but they need to be gc'd before the xcb connection is disconnected. Because it needs to come before the disconnect call, there isn't a straightforward weakref solution, and you haven't set any precedent for putting something like a `.destroy()` method on the XCBSurface, so I wasn't sure if that should be added.